### PR TITLE
feat (docs): install bakery onprem on azure vm

### PR DESCRIPTION
**Pull request by AlexandrStadolnik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +79,57 @@ This document describes how to deploy ui-bakery on-prem via `install.sh` script.
 
 **NOTE**: If Docker of the version less than the required (minimum 20.10.11) is already installed on the server, and/or Docker Compose (minimum 1.29.2), the script will be stopped. You need to update the versions of components manually and run the script again.
 
+#### Features of installation on Azure Virtual machine with Ubuntu
```
Comment (Tibing) [inline README.md:82]:
```suggestion
## Deploying on Azure Virtual Machine```

File: README.md [hunk 2]
```diff
@@ -78,6 +79,57 @@ This document describes how to deploy ui-bakery on-prem via `install.sh` script.
...
+
+1. Open [Azure Portal](http://portal.azure.com/) and on the search field in the top type in `Virtual machines` and select this item in result search window
+
+1. Click on `+ Create` button in the top left corner and select `Azure Virtual machine` in the menu
+
+1. Select an image of `Ubuntu 18.04` or higher
+
+1. For instance size, select minimum `Standard_F2s_v2 - 2 vCPUs, 4 GiB memory`
+
+1. If you want to access through password, in `Administrator account` section in `Authentication type` menu item select `Password` and setup admin username and password. Otherwise leave `SSH public key` and it will generate the keys after creation
```
Comment (Tibing) [inline README.md:92]: Let's keep only one straightforward way - SSH keys

File: README.md [hunk 3]
```diff
@@ -78,6 +79,57 @@ This document describes how to deploy ui-bakery on-prem via `install.sh` script.
...
+
+1. In `Inbound port rules` leave `SSH(22)` as by default
```
Comment (Tibing) [inline README.md:94]: Don't you think we have to open other ports to allow http/https traffic?

File: README.md [hunk 4]
```diff
@@ -78,6 +79,57 @@ This document describes how to deploy ui-bakery on-prem via `install.sh` script.
...
+
+1. In the `Networking` tab select existing or create new Virtual Network and Subnet used by this VM
+
+1. Public IP address will be created automatically
```
Comment (Tibing) [inline README.md:98]: I don't think we need this line here

File: README.md [hunk 5]
```diff
@@ -78,6 +79,57 @@ This document describes how to deploy ui-bakery on-prem via `install.sh` script.
...
+
+1. Check that `SSH (TCP/22)` is confugured by default. If not, add it manually using same algorithm as for {BakeryPort}
+
+1. Click `Ok` to finish creating Network security group
+
+1. Click `Review + create`
+
+1. Click `Create`
+
+1. After creating and running the virtual machine, connect to it from outside using admin/password or the SSH protocol, as it was configured.
```
Comment (Tibing) [inline README.md:120]: Let's keep only one straightforward way - SSH keys

File: README.md [hunk 6]
```diff
@@ -78,6 +79,57 @@ This document describes how to deploy ui-bakery on-prem via `install.sh` script.
...
+
+1. Run this command preferably from `/home` Linux directory to download, install and launch UI Bakery:
+
+   ```bash
+   curl -k -L -o install.sh https://raw.githubusercontent.com/uibakery/self-hosted/main/install.sh && bash ./install.sh
+   ```
+
+1. In the process, upon request, enter the previously received license code, hosting URL and port.
```
Comment (Tibing) [inline README.md:128]: We don't need to enter port here I bet. Becaseu we've already opened :3030

Comment (Tibing) [reply README.md:128]: also let's tell where to get hosting URL - it'll be the IP address of the VM

File: README.md [hunk 7]
```diff
@@ -78,6 +79,57 @@ This document describes how to deploy ui-bakery on-prem via `install.sh` script.
...
+
+1. After the installation is completed and launched, enter the bakery from a browser on your local machine at
+   (http://{Public IP address Azure VM}:{BakeryPort}
```
Comment (Tibing) [inline README.md:131]:
```suggestion
```

File: README.md [hunk 8]
```diff
@@ -78,6 +79,57 @@ This document describes how to deploy ui-bakery on-prem via `install.sh` script.
...
+
+1. Run this command preferably from `/home` Linux directory to download, install and launch UI Bakery:
+
+   ```bash
+   curl -k -L -o install.sh https://raw.githubusercontent.com/uibakery/self-hosted/main/install.sh && bash ./install.sh
+   ```
+
+1. In the process, upon request, enter the previously received license code, hosting URL and port.
+
+1. After the installation is completed and launched, enter the bakery from a browser on your local machine at
```
Comment (Tibing) [inline README.md:130]:
```suggestion
1. After the installation is completed and launched, enter the bakery from a browser on your local machine at http://{Public IP address of Azure VM}:3030
```

**Review by Tibing — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (AlexandrStadolnik) [reply README.md:94]: I tested it with only Bakery and SSH ports open and it worked

File: README.md (hunk 2 above)
Comment (AlexandrStadolnik) [reply README.md]: Done

File: README.md (hunk 4 above)
Comment (AlexandrStadolnik) [reply README.md]: Done

File: README.md (hunk 5 above)
Comment (AlexandrStadolnik) [reply README.md]: Done

File: README.md (hunk 6 above)
Comment (AlexandrStadolnik) [reply README.md]: We need to enter the port, because it is used now for Bakery installation - same as on local machine

File: README.md (hunk 7 above)
Comment (AlexandrStadolnik) [reply README.md]: Done

File: README.md (hunk 8 above)
Comment (AlexandrStadolnik) [reply README.md]: Done

File: README.md (hunk 6 above)
Comment (AlexandrStadolnik) [reply README.md]: Fixed

**Review by Tibing — APPROVED** [review]

**Pull request merged**